coeus-impl/src/main/java/org/kuali/coeus/common/budget/framework/personnel/BudgetPersonnelDetails.java

```diff
@@ -636,7 +636,7 @@ public void setLineItemSequence(Integer lineItemSequence) {
 
     @Override
     public CostElement getCostElementBO() {
-        return costElementBO;
```

Comment:
please remove the costElementBO since it is transient and seems to serve no useful purpose